jcl/src/jdk.jcmd/share/classes/openj9/tools/attach/diagnostics/tools/Jcmd.java

```diff
@@ -65,47 +67,61 @@ public class Jcmd {
 	 * @param args Application arguments.  See help text for details.
 	 */
 	public static void main(String[] args) {
-		String command = null;
-		/* An empty argument list is a request for a list of VMs */
+		if ((args.length == 1) && Arrays.stream(HELP_OPTIONS).anyMatch(args[0]::equals)) {
+			System.out.printf(HELPTEXT);
+			return;
+		}
+
+		List<AttachProvider> providers = AttachProvider.providers();
+		if ((providers == null) || providers.isEmpty()) {
+			System.err.println("no attach providers available"); //$NON-NLS-1$
+			return;
+		}
+
+		AttachProvider openj9Provider = providers.get(0);
+		List<VirtualMachineDescriptor> vmds = openj9Provider.listVirtualMachines();
+		if ((vmds == null) || vmds.isEmpty()) {
+			System.err.println("no vm found"); //$NON-NLS-1$
```

Comment:
vm -> VMs (it should be plural to match the context)
```java
			System.err.println("no VMs found"); //$NON-NLS-1$
```

Reply:
Updated